aerogearcatalog/metrics-apb
# AEROGEAR-2701 framework field

Updates existing dashboards with recently added framework field.
JIRA: https://issues.jboss.org/browse/AEROGEAR-2701

How to verify:

* Provision metrics apb from this branch
* It seems https://github.com/aerogear/aerogear-app-metrics/pull/56 is not published on Docker hub.
  * Check it out locally
  * Build it and push it to your Docker org
  * Edit deployment config for the app metrics pod and use your tag for the image
* Use https://github.com/aliok/aerogear-mobile-metrics-api-data-generator to generate data (check readme)
* Check the dashboards. Play with the "Framework" dropdown at the top of mobile app metrics and security metrics dashboards
